website/routes.py

```diff
@@ -580,6 +580,28 @@ def make_url_map(app):
             OsfWebRenderer('profile/oauth_app_detail.mako', trust=False)
         ),
 
+        Rule(
+            '/settings/tokens/',
```

Comment:
Standard confirmation: all new routes should be approved by routing Tzars Brian Geiger and/or Jeff Spies. Can you post a confirmation if/when these routes have been approved?